Firestore/core/test/firebase/firestore/local/leveldb_opener_test.cc

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright 2019 Google
```

Comment:
nit: since you updated the year to 2020 on the others, this one too?  (same for the other added files in this PR)